### PR TITLE
TECH Add development var to ignore auth

### PR DESCRIPTION
**Pull request by anthonyolea**
Add an environment variable `IGNORE_AUTH` to explicitly bypass the authentication and the role authorization process for development purpose only

### CODE REVIEW

**Comment by anthonyolea** [comment]
During development, for testing purpose, you want an hassle-free context to test your API.
Authentication requires both the public key for the server-side and the corresponding valid token for the client-side.

**Comment by jean-k-cp** [comment]
@ltbesh-cp we already do this in some other (former Newton) projects like Gaia to run a dev server:
https://github.com/transcovo/gaia

I think it would be beneficial for all further micro-services in go to share this feature with a consistent variable naming, this is why I proposed to add it directly in the middleware lib. This is of course subject to debate, so let's debate:

File: jwtAuth.go [hunk 1]
```diff
@@ -98,11 +96,12 @@ Panics if fails to parse the public key
 func JwtAuthenticationMiddleware(publicKeyString string, logger *logrus.Logger) Middleware {
 	/*
 		If the IGNORE_AUTH environment variable is set to "true"
-		the middleware will bypass the authentication process
+		the middleware will bypass the authentication and authorization process
 		/!\ This variable should be set to true only for development purpose /!\
```
Comment (jean-k-cp) [inline jwtAuth.go]: this explanation belongs to the doc, ie the README

Comment (anthonyolea) [inline jwtAuth.go]: Fixed

**Pull request merged**